src/mixins/filing-template-mixin.ts

```diff
@@ -766,7 +766,12 @@ export default class FilingTemplateMixin extends DateMixin {
 
     if (this.isCoopCorrectionFiling) {
       this.storeSpecialResolutionRulesAndMemorandum(filing.correction, entitySnapshot)
-      const specialResolution = this.getLatestResolutionForBusiness
+      let specialResolution: SpecialResolutionIF = {}
+      if (filing.correction.resolution) {
+        specialResolution = filing.correction
```

Comment:
Need for for draft

Reply:
It seems weird to assign the entire `correction` object to `specialResolution`. I thought `correction` would contain `specialResolution` (or some of its properties), but this makes them seem equivalent...?

Should it look like this?
```
specialResolution = {
  meetingDate: filing.correction.meetingDate,
  resolution: filing.correction.resolution,
  resolutionDate: filing.correction.resolutionDate,
  <etc>
}
```

Reply:
By the way, I think there's a typing error on line 145.

Reply:
yes correction includes much more properties, I'll update it